public/employers/pak-a-sak.yml

```diff
@@ -0,0 +1,20 @@
+name: Pak-A-Sak # Of Indiana fame, NOT the one in Texas.
+image: pak-a-sak.svg#49a1d1
```

Comment:
You can add for this Pak-a-Sak "Jay Petroleum" as part of its alias. I think the major difference is this PAS Indiana has gas stations while the other one doesn't.